tests/chainer_tests/links_tests/connection_tests/test_n_step_lstm.py

```diff
@@ -15,7 +15,6 @@ def sigmoid(x):
 
 
 @testing.parameterize(*testing.product({
-    'use_cudnn': ['always', 'auto', 'never'],
```

Comment:
This test case is redundant. 
`tests/chainer_tests/functionss_tests/connection_tests/test_n_step_lstm.py` is covered this case.